parts/robot.sh

```diff
@@ -14,6 +14,22 @@ apt-get install -y $PACKAGES_DIR/*.deb
 apt-get install -y udiskie
 cp /tmp/packer-files/udiskie/udiskie.yml /etc/
 cp /tmp/packer-files/udiskie/udiskie.service /lib/systemd/system/
+systemctl enable udiskie.service
+
+# Configure udisks2 to allow mounting as other users
+mkdir -p /etc/udisks2
+cat >/etc/udisks2/mount_options.conf <<EOF
```

Comment:
Suggestion: Instead, what if we keep the file as its own file, and inject variables as needed with `envsubst`? Might make the files easier to maintain

Reply:
The variables you need to populate in the file are so tightly tied to here that this is likely to be harder to maintain as you need to update two locations